python_modules/libraries/dagster-dbt/dagster_dbt/components/dbt_project/component.py

```diff
@@ -128,7 +128,9 @@ class DbtProjectComponent(Component, Resolvable):
     translation: Optional[ResolvedTranslationFn] = None
     select: str = "fqn:*"
     exclude: Optional[str] = None
-    translation_settings: Optional[DagsterDbtTranslatorSettings] = None
+    translation_settings: Optional[DagsterDbtTranslatorSettings] = field(
+        default_factory=lambda: DagsterDbtTranslatorSettings(enable_code_references=True)
+    )
```

Comment:
i think the behavior here might be rough if someone is setting other settings on the the object or in yaml and then getting the object default of `enable_code_references=False`